packages/eslint-config-sentry-app/index.js

```diff
@@ -72,3 +74,24 @@ module.exports = {
     'sentry/no-digits-in-tn': ['error'],
   },
 };
+
+//reuses the common rule import/order from the package 'eslint-config-sentry-react'
+config.rules['import/order'] = eslintConfigSentryReact.rules['import/order']
+
+// and includes a sentry/getSentry package-specific rule - pathGroups
+config.rules['import/order'][1] = {
+  ...config.rules['import/order'][1],
+  pathGroups: [
+    {
+      pattern: '@emotion/styled',
+      group: 'external',
+    },
+    {
+      pattern: 'sentry*/**',
+      group: 'internal',
```

Comment:
Should we add `position: before`?

Reply:
when I use this property an empty line is always added between `'sentry*/**' `and the `internal` imports..

Reply:
That might be nice to see in `getsentry`, but I don't have a strong opinion here

Reply:
please what do you think @getsentry/app-frontend ?

Reply:
@markstory any thoughts on what billy said about `getstring`?

Reply:
Do we have may 'sentry' imports outside of the browser SDK which to me is an 'external' import to the react app.

Reply:
As we already have `getSentry` alias in

`webpack.layer.js`
https://github.com/getsentry/getsentry/blob/868c932a71b677eab9cb5066ed1d7aba49f6328b/webpack.layer.js#L62

and in the `.eslintrc`

https://github.com/getsentry/getsentry/blob/868c932a71b677eab9cb5066ed1d7aba49f6328b/.eslintrc.js#L82

 I think it is unnecessary